gimme_aws_creds/main.py

```diff
@@ -185,7 +185,7 @@ def _get_partition_and_region_from_saml_acs(saml_acs_url):
             raise errors.GimmeAWSCredsError("{} is an unknown ACS URL".format(saml_acs_url))
 
     @staticmethod
-    def _get_sts_creds(partition, region, assertion, idp, role, duration=3600):
+    def _get_sts_creds(partition, region, assertion, idp, role, duration=3600, session_name="default-session"):
```

Comment:
I guess for traceability it's best to set default session name to a more `gimme-aws-creds`-related value just in case.